front/src/applications/operationalStudies/hooks/useSetupItineraryForTrainUpdate.ts

```diff
@@ -237,32 +237,36 @@ const useSetupItineraryForTrainUpdate = (
         return;
       }
       const trainSchedule = await getTrainScheduleById({ id: trainIdToEdit }).unwrap();
-      if (!trainSchedule.rolling_stock_name) {
-        return;
-      }
 
       let itinerary = null;
-      try {
-        const rollingStock = await getRollingStockByName({
-          rollingStockName: trainSchedule.rolling_stock_name,
-        }).unwrap();
-        itinerary = await computeItineraryForTrainUpdate(trainSchedule, rollingStock);
-        const { pathSteps, pathProperties } = itinerary || {};
+      let rollingStock: RollingStockWithLiveries | null = null;
+      let pathSteps: (PathStep | null)[] | undefined;
```

Comment:
Nit: we don't really need this variable. In the `adjustConfWithTrainToModifyV2()` call we can use `itinerary?.pathSteps` instead.

Reply:
Yes you're right, I remove it :+1: